admin/services/class-indexable.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Services
+ */
+
+/**
+ * Represents the indexable service.
+ */
+class WPSEO_Indexable_Service {
+
+	/**
+	 * Retrieves an indexable.
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 *
+	 * @return WP_REST_Response The response.
+	 */
+	public function get_indexable( WP_REST_Request $request ) {
+
+		$object_type = $request->get_param( 'object_type' );
+		$provider = $this->get_provider( $object_type );
+
+		if ( $provider === null ) {
+			return new WP_REST_Response( 'Unknown type ' . $object_type, 404 );
+		}
+
+		$object_id = $request->get_param( 'object_id' );
+		if ( ! $provider->is_indexable( $object_id ) ) {
+			return new WP_REST_Response( 'Object with id ' . $object_id . ' not found', 404 );
```

Comment:
We know the type here, so instead of `Object with id` it can be `$object_type with ID ...`
This string can be translated, so that should be added.